src/Microsoft.SqlTools.ServiceLayer/EditData/EditDataService.cs

```diff
@@ -79,6 +79,7 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(EditDeleteRowRequest.Type, HandleDeleteRowRequest);
             serviceHost.SetRequestHandler(EditDisposeRequest.Type, HandleDisposeRequest);
             serviceHost.SetRequestHandler(EditInitializeRequest.Type, HandleInitializeRequest);
+            serviceHost.SetRequestHandler(EditRevertCellRequest.Type, HandleRevertCellRequest);
```

Comment:
do we have a use case for this message type?  I was suspecting that the grid would do reverts on the row level instead of the cell level.